compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -279,6 +279,8 @@ message ScenarioError {
     Empty ComparableValueError = 29;
     ContractIdInContractKey contract_id_in_contract_key = 30;
     Empty ValueExceedsMaxNesting = 31;
+    string scenario_user_not_found = 33;
```

Comment:
is it worth grouping them in the protobuf into a oneof just for user management? don’t have a strong opinion here so go with what you prefer.

Reply:
I think _flat_ better matches the style of the existing proto error cases.